cdc/capture/capture.go

```diff
@@ -326,15 +315,13 @@ func (c *captureImpl) run(stdCtx context.Context) error {
 		// an unrecoverable serious errors (recoverable errors are intercepted in the owner tick)
 		// so we should restart the capture.
 		err := c.campaignOwner(ctx)
-		if err != nil {
+		if err != nil || c.liveness.Load() != model.LivenessCaptureStopping {
 			log.Error("campaign owner routine exited with error, restart the capture",
```

Comment:
the log  is not correct when `c.liveness.Load() != model.LivenessCaptureStopping `